server/plugin.go

```diff
@@ -279,7 +280,36 @@ func (p *Plugin) Send(link ChannelLink, user *model.User, post *model.Post) (str
 		parentID, _ = p.API.KVGet(mattermostTeamsPostKey(post.RootId))
 	}
 
-	newMessageId, err := p.msteamsBotClient.SendMessage(link.MSTeamsTeam, link.MSTeamsChannel, string(parentID), text)
+	var attachments []*msteams.Attachment
+	for _, fileId := range post.FileIds {
+		fileInfo, appErr := p.API.GetFileInfo(fileId)
+		if appErr != nil {
+			p.API.LogError("Unable to get file attachment", "error", appErr)
```

Comment:
Should be `LogWarn` if it's not something an admin can fix. Same with other instances of this in the function/PR